gen-ai/inference/vllm-rayserve-inf2/ray-service.yaml

```diff
@@ -0,0 +1,153 @@
+#----------------------------------------------------------------------
+# NOTE: For deployment instructions, refer to the DoEKS website.
+#----------------------------------------------------------------------
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: vllm
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hf-token
+  namespace: vllm
+data:
+  hf-token: $HUGGING_FACE_HUB_TOKEN
+---
+apiVersion: ray.io/v1
```

Comment:
I think we want to be a little more consistent with how we set up the resource configuration. We have 3 max replicas. We should be consistent with deciding if we want to scale nodes or if we want to scale actors. If we want to allow for intra node scaling, I believe you want to set the 
```
neuron_codes = (6 * 2) / 3
```
 or 
```
(NUM_NEURON_DEVICES * 2) / NUM_REPLICAS_PER_NODE
```
We also want to make sure we set the `num_cpus = total_node_cpus / num_replicas`